stable/drone/templates/deployment-agent.yaml

```diff
@@ -54,6 +58,28 @@ spec:
           - name: {{ $key }}
             value: {{ $value | quote }}
           {{- end }}
+{{- if .Values.agent.livenessProbe.enabled }}
+        livenessProbe:
+          httpGet:
+            path: "/healthz"
+            port: http
+          initialDelaySeconds: {{ .Values.agent.livenessProbe.initialDelaySeconds }}
```

Comment:
Please add these new configs also in the README.md

Reply:
Done.